core/iwasm/compilation/aot_llvm_extra.cpp

```diff
@@ -66,6 +72,10 @@ aot_add_simple_loop_unswitch_pass(LLVMPassManagerRef pass);
 void
 aot_apply_llvm_new_pass_manager(AOTCompContext *comp_ctx, LLVMModuleRef module);
 
+void
+wasm_dpgo_set_prof_meta(AOTCompContext *comp_ctx, LLVMValueRef function,
+                        uint32_t func_idx);
```

Comment:
Had better declare it in `aot_llvm.h` or `wasm.h`?

Reply:
move dpgo related implementation aot_dpgo.h and aot_dpgo.cpp